src/main/resources/application.yml

```diff
@@ -1,14 +1,22 @@
+server:
+  port: 8082
 spring:
+  application:
+    name: hospital-finance
   datasource:
-    url: jdbc:postgresql://localhost:5432/postgres?currentSchema=finance
+    url: jdbc:postgresql://${DATASOURCE_HOST}/postgres?currentSchema=finance
     username: ${DATASOURCE_USERNAME}
     password: ${DATASOURCE_PASSWORD}
     driver-class-name: org.postgresql.Driver
   jpa:
     database: postgresql
     database-platform: org.hibernate.dialect.PostgreSQLDialect
-#    hibernate:
-#      ddl-auto: create
+    hibernate:
+      ddl-auto: update
```

Comment:
better to use some sort of schema version control like ```Liquibase``` instead of letting ORM control schema